src/lib/env.js

```diff
@@ -12,12 +12,13 @@ const parsers = {
 };
 
 export class Env {
-  constructor(defaultLang, fetchResource) {
-    this.defaultLang = defaultLang;
+  constructor(fetchResource) {
     this.fetchResource = fetchResource;
 
     this._resLists = new Map();
     this._resCache = new Map();
+    // XXX remove when sw land
+    this._appVersions = new WeakMap();
```

Comment:
@zbraniecki I'm not sure if this should be defined here, or maybe I could store the `appVersion` on the `langs` objects, where we already have the language code, direction and source (`app` or `extra` for langpacks).  wdyt?